src/contact.js

```diff
@@ -0,0 +1,57 @@
+import './contact.css';
+import Maps from './Images/map.png';
+
+function component() {
+  const element = document.getElementById('contact');
+  const header = document.createElement('header');
+  const nav = document.createElement('nav');
+  const home = document.createElement('a');
+  const menu = document.createElement('a');
+  const contact = document.createElement('a');
+  const logo = document.createElement('a');
+  const section = document.createElement('section');
```

Comment:
- [ ] Please change this and all other regular functions to es6 arrow function syntax.

Reply:
> * [ ]  Please change this and all other regular functions to es6 arrow function syntax.

- [ ] REQUIRED - As requested previously, please change this function and all other similar regular function syntaxes throughout the code into es6 arrow function syntax. It's important the code keeps up to date with recent trends.